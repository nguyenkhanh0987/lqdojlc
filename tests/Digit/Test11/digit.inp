100
938566204 981307540
299925689 582582392
766417087 953832017
680835897 810322883
292289606 610629644
661525711 964513686
586791930 943009419
573186331 695928127
196622588 234668487
269619910 987776684
505474881 794822896
811243329 987558849
168208981 385657203
601576766 898839549
557496008 628027372
19835211 608979773
14310163 274398846
84526040 180671966
877205886 922227533
35348563 526191264
533468488 754562950
228243670 289491009
247995648 283387642
206407522 273322934
28811697 607974564
491386217 541177190
10526980 744300268
394025350 725056787
624955061 853493331
376823498 882857058
356934249 519077274
347464505 594046470
269828199 898228141
683461348 770644240
123753721 173082515
517041676 521434209
266933753 903709982
269463012 902810703
690069064 976274326
742392842 945736917
327716624 460576210
865210352 873740001
87561634 569014523
300038215 693985186
336012941 905876961
109424177 381783381
708411199 722155821
183337862 570326457
505574397 624968384
940745002 962641718
92198269 560869778
870507208 977419181
937070209 943467748
140365003 594173665
314920264 934417428
502489207 900344963
163204217 543409239
101379583 167476448
445128089 462037100
652794306 854042879
282240692 417421483
661187221 700419077
858565847 932306189
159403396 161804469
25739879 307769840
214660115 860042633
535658139 740586325
175605015 704079368
384559307 936617928
883474575 942429137
224972271 471447906
458834703 553595911
866076388 944341677
712237211 978770733
115330849 126524035
111161159 855532410
2109133 823536496
259703462 588399248
80082637 730444194
435808197 536603670
730168104 880928359
26259814 93256244
150707792 974607025
969200425 986455550
434387248 452558157
329796504 803658729
929291176 989266084
126000102 657058491
291676671 900189944
228017486 585034154
819562990 961086181
400472302 502364266
993544641 999874122
952185791 964531934
97826080 417879604
557912095 867742220
146456827 282362283
104104236 535183690
970685138 993115207
724852603 942156340
