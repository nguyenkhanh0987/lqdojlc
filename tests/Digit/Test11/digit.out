2914042
37740912
13619421
12136761
43229769
24333679
29840055
14463983
4824551
79033438
29763340
11607102
37652715
25937251
8472230
96236431
49451738
18297057
3299683
82819800
22838262
11551883
7255431
10279578
94689016
4715798
109588397
37941638
19921554
54584748
23466127
34857295
74217820
7178808
9313360
238701
74825145
74485913
21290097
15325112
18932422
822179
75355658
50290645
63576862
46821327
582546
57158313
12782769
1329470
72920192
7250161
309650
71098111
66631555
37783724
58708650
10789089
2292791
17773796
21033555
5883301
5455490
620223
54419774
78223656
21383663
74878441
56014125
3276423
37882055
12161759
5235451
19141620
1485181
97542101
117693002
48052591
94465619
13140919
12320509
15324955
99651260
1755112
2154734
57304770
4070948
78036758
69127164
52518851
9014464
13792090
582306
846946
53745013
29362627
25488531
67386345
2426812
16140658
